src/components/Laatija/LaatijaForm.svelte

```diff
@@ -165,7 +164,8 @@
           parse={formParsers.sukunimi}
           validators={formSchema.sukunimi}
           disabled={!isPaakayttaja}
-          i18n={$_} />
+          i18n={$_}
+        />
```

Comment:
miksi toi formatointi heittää tämän eri riville

Reply:
Mulla formatointi ei tee näin. Mitä editoria käytät?

Reply:
VS Code